packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -0,0 +1,190 @@
+import React, { Fragment } from 'react';
+
+import { useCheckoutPage } from '@magento/peregrine/lib/talons/CheckoutPage/useCheckoutPage';
+
+import { Title } from '../../components/Head';
+import Button from '../Button';
+import PriceSummary from './PriceSummary';
+import PriceAdjustments from './PriceAdjustments';
+import PaymentInformation from './PaymentInformation';
+import ShippingMethod from './ShippingMethod';
+import ShippingInformation from './ShippingInformation';
+import ItemsReview from './ItemsReview';
+import OrderConfirmationPage from './OrderConfirmationPage';
+import CREATE_CART_MUTATION from '../../queries/createCart.graphql';
+import GET_CART_DETAILS_QUERY from '../../queries/getCartDetails.graphql';
+
+import { mergeClasses } from '../../classify';
+
+import defaultClasses from './checkoutPage.css';
+
+const GuestCheckoutOptions = props => {
+    const { classes, handleSignIn } = props;
+
+    return (
+        <div className={classes.signin_container}>
+            <Button
+                className={classes.sign_in}
+                onClick={handleSignIn}
+                priority="high"
+            >
+                {'Login and Checkout Faster'}
+            </Button>
+        </div>
+    );
+};
+
+const GreetingMessage = props => {
+    const { isGuestCheckout, classes } = props;
+
+    return (
+        <div className={classes.heading_container}>
+            <h1 className={classes.heading}>
+                {isGuestCheckout ? 'Guest Checkout' : 'Review and Place Order'}
+            </h1>
+        </div>
+    );
+};
+
+const EmptyCartMessage = props => {
+    const { classes, isGuestCheckout } = props;
+
+    return (
+        <div className={classes.signin_container}>
+            <div className={classes.heading_container}>
+                <h1 className={classes.heading}>
+                    {isGuestCheckout ? 'Guest Checkout' : 'Checkout'}
+                </h1>
+            </div>
+            <h3>There are no items in your cart.</h3>
```

Comment:
Empty cart feels sparse. Surprised there isn't a call to action like "Continue Shopping".

[![Image from Gyazo](https://i.gyazo.com/a5259e16a785d23299ab460654869f14.png)](https://gyazo.com/a5259e16a785d23299ab460654869f14)

Reply:
This was never part of the XD or the epic. So I added filler text. UX needs to come up with a design for this.

@soumya-ashok @schensley drop your thoughts here.